scripts/resources/tachidesk-electron-launcher-aur.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if [ ! -f /usr/bin/electron ]; then
+	echo "Electron executable was not found! In order to run this launcher, you need Electron installed."
```

Comment:
maybe mention the actual package name and an instruction to install electron with apt?

Reply:
I couldn't find electron in Debian or Ubuntu's repositories. They install it with npm

Reply:
It will be like this:
```
sudo apt install nodejs npm
npm install electron -g
```

Reply:
then ship it with the deb package?

Reply:
Maybe someone doesn't want to use Electron launcher and download an additional 100MBs